src/l1-resources/cluster-outage-simulation/README.md

```diff
@@ -0,0 +1,71 @@
+# cloud-outage-simulation
+
+The official [MongoDB Atlas](https://www.mongodb.com/) AWS CDK resource for Node.js.
+
+> AWS CDK [L1 construct] and data structures for the [AWS CloudFormation Registry] type `MongoDB::Atlas::ClusterOutageSimulation` v1.0.0.
```

Comment:
What does this version (`v1.0.0`) correspond to? Because if it associated to the CFN resource, I imagine it will depend on which extension you activate and the region right?